src/main/java/com/o19s/es/ltr/query/StoredLtrQueryBuilder.java

```diff
@@ -67,9 +68,17 @@
         declareStandardFields(PARSER);
     }
 
-    public StoredLtrQueryBuilder() {}
-    public StoredLtrQueryBuilder(StreamInput input) throws IOException {
+    public StoredLtrQueryBuilder() {
```

Comment:
Is it really only for testing? If that's the case, wouldn't it make more sense to inject this fake store loader in the test? Or at least tag this constructor with `@VisibleForTesting`?